### ISSUE TITLE
Voices show up in the control panel, but they aren't used.

### ISSUE THREAD

**Issue by djkacevedo** [awslabs/amazon-polly-tts-for-windows#8]
Voices show in the control panel to be selected, and they play the test there fine, but when I use an application that uses the voice, it either gets set back to Windows David, or the application crashes.

Specifically, the Kindle app for PC sets it to David, while Brave and Chrome crash.

Polly Player works fine though.

**Comment by troylar** [comment]
How are you using the voices in Brave/Chome?

**Comment by MikeDrones** [comment]
yes i have the same issue, polly voices show up under "Control Panel\Ease of Access,  text to speech settings" i am able to select them and preview but under settings>voice>choose a voice, polly is not available. 

in an application for example adobe i have "default reader" selected. 
it uses the windows voice.

**Comment by heldiho53** [comment]
This is a very serious issue, making this completely unusable, any updates on this @troylar ?
